content/en/docs/ambient/getting-started/cleanup/index.md

```diff
@@ -1,4 +1,3 @@
-
 ---
 title: Cleanup
```

Comment:
oh, this en file shall be removed from your commit before zh-approvers to merge this.

Reply:
Sure, let's fix en typo in other PR.